api/interestr/api/models.py

```diff
@@ -65,17 +65,23 @@ def get_picture(self):
         else:
             return '/static/assets/img/group_default_icon.png'
 
-
 class Post(BaseModel):
     owner = models.ForeignKey(auth_models.User, related_name="posts", default=None, null=True)
     text = models.TextField(default='')
     group = models.ForeignKey(Group, related_name='posts', on_delete=models.CASCADE, default=None, null=True)
     data_template = models.ForeignKey('api.DataTemplate', related_name='posts', default=None, null=True)
     data = JSONField(default=None, null=True)
-
+     
     def __str__(self):
         return self.text
 
+class Comment(BaseModel):
+    owner = models.ForeignKey(auth_models.User, related_name="comments", default=None, null=True)
```

Comment:
I think comment owner field should not allow null values. A comment should not be without an owner.

Reply:
To be honest, since it was done similarly for the owner and group fields of the post I assumed that I was missing some subtle point connected with keeping the instance after deletion of the user. Now I realize that for this we should overwrite the "on_delete" attribute of the ForeignKey field. Which way we should go, am I missing something?

Reply:
Really! Interesting. We should change the post model too then. on_delete is cascade, by default. So, it doesn't have to be changed.